.github/workflows/ci.yml

```diff
@@ -45,7 +48,7 @@ jobs:
     # Only run tests in the architecture that the runners are using
     - name: Run ${{ matrix.channel }} tests
       run: cargo +${{ matrix.channel }} test --target ${{ matrix.target }} ${{ matrix.features }} --verbose
-      if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' }}
+      if: ${{ contains(matrix.target, 'x86') }}
```

Comment:
I think this'll probably need to be either an exact comparison or at least a check for the substrings `x86_64` and `i686`.

Reply:
Yes I just saw your response, will update